app/src/main/java/com/example/util/simpletimetracker/wear/DomainAPI.kt

```diff
@@ -0,0 +1,76 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+package com.example.util.simpletimetracker.wear
+
+import com.example.util.simpletimetracker.domain.interactor.PrefsInteractor
+import com.example.util.simpletimetracker.domain.interactor.RecordTagInteractor
+import com.example.util.simpletimetracker.domain.interactor.RecordTypeInteractor
+import com.example.util.simpletimetracker.domain.interactor.RunningRecordInteractor
+import com.example.util.simpletimetracker.domain.mapper.AppColorMapper
+import com.example.util.simpletimetracker.domain.model.RunningRecord
+import com.example.util.simpletimetracker.wearrpc.Activity
+import com.example.util.simpletimetracker.wearrpc.CurrentActivity
+import com.example.util.simpletimetracker.wearrpc.Settings
+import com.example.util.simpletimetracker.wearrpc.SimpleTimeTrackerAPI
+import com.example.util.simpletimetracker.wearrpc.Tag
+
+class DomainAPI(
+    private val prefsInteractor: PrefsInteractor,
+    private val recordTypeInteractor: RecordTypeInteractor,
+    private val recordTagInteractor: RecordTagInteractor,
+    private val runningRecordInteractor: RunningRecordInteractor,
+    private val appColorMapper: AppColorMapper,
+) : SimpleTimeTrackerAPI {
+
+    override suspend fun queryActivities(): Array<Activity> {
+        return recordTypeInteractor.getAll().filter { recordType -> !recordType.hidden }
+            .map { recordType ->
+                val color = appColorMapper.mapToColorInt(recordType.color)
+                val hex = String.format("#%06X", (0xFFFFFF and color))
```

Comment:
I opted for the hex code since it is...
1. More useful to view in debug logs of the communication AND
2. Easier to verify against the hex code in the phone app's UI

If you have a strong preference otherwise, it will work fine to use the int instead.

Reply:
Better to use color int.